ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/host1.cfg

```diff
@@ -2850,6 +2850,7 @@ interface Vxlan1
    vxlan flood vtep learned data-plane
    vxlan vlan 110 vni 10110
    vxlan vlan 111 vni 10111
+   vxlan vlan 113,115-118 vni 10113,10115-10118
```

Comment:
Note: 
If the ranges overlap, it throws an error in EOS, where as our code generates the command-
```
s1-leaf2(config-if-Vx1)#vxlan vlan 112 vni 10112
s1-leaf2(config-if-Vx1)#vxlan vlan 108-114 vni 10108-10114
% VLAN to VNI mapping already exists. Use 'add/remove' keyword to modify existing map
```

Reply:
You have to _either_ use the range form or the non-range form.
The error here is no different than someone doing `vlan 110 vni 10110` and `vlan 111 vni 10110` which we are not catching either.
I don't think we need to change anything for this.